base/applications/regedit/main.c

```diff
@@ -205,6 +216,8 @@ int WINAPI wWinMain(HINSTANCE hInstance,
 
     UNREFERENCED_PARAMETER(hPrevInstance);
 
+    OleInitialize(0);
```

Comment:
```suggestion
    OleInitialize(NULL);
```
(parameter is pointer-type)